packages/xrpl/test/models/CredentialAccept.test.ts

```
@@ -0,0 +1,150 @@
+      Sequence: 1337,
+      Flags: 0,
+    } as any
+  })
+
+  it(`verifies valid CredentialAccept`, function () {
+    assert.doesNotThrow(() => validateCredentialAccept(credentialAccept))
+    assert.doesNotThrow(() => validate(credentialAccept))
+  })
+
+  it(`throws w/ missing field Account`, function () {
+    delete credentialAccept.Issuer
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)